src/components/all-articles.jsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from "react";
+import '../styling/article.styling.css'
+
+const AllArticles = () => {
+    const [articles, setArticles] = useState([])
+    // const [sortType, setSortType] = useState('created_at');
+
+    useEffect(() => {
+        fetch('https://completed-nc-news-api.herokuapp.com/api/articles')
+        .then(response => response.json())
+        .then(data => setArticles(data.articles))
+
+        // const sortArray = type => {
+        //     const types = {
+        //         created_at: 'created_at',
+        //         votes: 'votes',
+        //     };
+        //     const sortProperty = types[type];
+        //     const sorted = [...articles].sort((a, b) => b[sortProperty] - a[sortProperty]);
+        //     setArticles(sorted);
+        // };
+        // sortArray(sortType);
+    }, []); 
+
+
+    return (
+        <>
+            <ul className="cards">
+                {articles.map((article) => {
+                    let strDate = `${article.created_at}`;
+                    let date = strDate.substring(0, 10);
+                    console.log(date)
```

Comment:
remove any console logs that you don't need anymore 😊

Reply:
will do!